### ISSUE TITLE
Coinfinder halting at ggtree

### ISSUE THREAD

**Issue by hannahpugh** [fwhelan/coinfinder#19]
Hi there,
I've tried running coinfinder on roary output from 187 assemblies of 38 E. coli STs.
It all seems to run fine but errors when in gets to the heatmap stage.
Then I get this....
"Calculating lineage dependence...
Generate coincidence heatmap...
ERROR MESSAGE FROM R:
Error: package or namespace load failed for ‘ggtree’:
 object ‘as_data_frame’ is not exported by 'namespace:tidytree'
Execution halted"

I have checked that all dependencies are there and current. It was downloaded with conda, so should be okay... Wasn't sure if it could be an R package/version issue? 
I don't know enough about this stuff, so not sure exactly how to fix. 
Any ideas?

The code I ran was this `coinfinder -i gene_presence_absence.csv -I -p coinfinder.nwk -v -x 4 -o Apr1_unrooted_testing -a`

Thanks

**Comment by fwhelan** [comment]
Hi Hannah,
Thanks for your question. I think this might be a problem with the version of some of the R dependencies that I updated in the last conda recipe, sorry about that. Would you be able to tell me what the output of the following are when you run R within your conda environment?
```
packageVersion("ggtree")
packageVersion("tidytree")
```
I've started some tests locally with my conda recipe and will let you know when I think I have a solution.
Cheers,
Fiona

**Comment by hannahpugh** [comment]
Hi there,
These are the versions I have installed.

```
> packageVersion("ggtree")
[1] ‘1.16.0’
> packageVersion("tidytree")
[1] ‘0.3.3’
```
Best,
Hannah

**Comment by fwhelan** [comment]
Hi Hannah,

I think I've narrowed this down to two bugs. I have released a new version (1.0.4) of coinfinder via conda which fixes one of them (how `network.R` assigns colours to nodes in the heatmap), and am currently trying to update my conda recipe to fix the second (forcing an updated version of `ggtree`). In the meantime, if you reinstalled coinfinder via conda and then update your `ggtree` package to version `2.0.0` manually, I think it should work smoothly. Alternatively, I'm hoping by the end of today that I'll have figured the second bug out and a clean conda install alone should do it; I'll comment here when I've got that working.

Cheers,
Fiona
